feature-libs/asm/root/services/asm-auth-http-header.service.spec.ts

```diff
@@ -52,6 +52,11 @@ class MockOccEndpointsService implements Partial<OccEndpointsService> {
   getBaseUrl() {
     return 'some-server/occ';
   }
+  getRawEndpointValue(endpoint: string): string {
```

Comment:
needs to be dded in asm spec file to avoid unit tests failures as also using auth-http-header.service.ts as super class
super.alterRequest(request, token);